pkg/xcp/xcp_test.go

```diff
@@ -44,6 +44,46 @@ func TestGetCIEnv(t *testing.T) {
 	os.Setenv("ATLANTIS_TERRAFORM_VERSION", "yyy")
 	os.Setenv("PULL_NUM", "yyy")
 	os.Setenv("REPO_REL_DIR", "yyy")
+	os.Setenv("BUILD_ID", "27")
+	os.Setenv("JOB_BASE_NAME", "main")
+	os.Setenv("KUBERNETES_PORT", "tcp://172.20.0.1:443")
+	os.Setenv("RUN_ARTIFACTS_DISPLAY_URL", "https://ci.intouchhealth.io/")
+	os.Setenv("TF_VAR_adminpassword", "****")
+	os.Setenv("TF_VAR_adminusername", "***")
+	os.Setenv("ARM_TENANT_ID", "****")
```

Comment:
nit: We don't have a test for the CLIENT substring

Reply:
sorry pushed the change. fixed it.